README.md

```diff
@@ -10,7 +10,7 @@ We prefix all our npm package names with `ml-` (eg. ml-matrix) so they are easy
 To include the ml.js library in a web page:
 
 ```html
-<script src="https://www.lactame.com/lib/ml/3.1.0/ml.min.js"></script>
+<script src="https://www.lactame.com/lib/ml/3.1.1/ml.min.js"></script>
```

Comment:
can you please revert this change? We update the version number here after the release.

Reply:
yep! updating the PR now